policy-engine/src/main.rs

```diff
@@ -0,0 +1,223 @@
+#[macro_use]
+extern crate log;
+#[macro_use]
+extern crate prometheus;
+
+mod scraper;
+mod policy;
+
+use actix::prelude::*;
+use actix_web::{web, App, HttpResponse};
+use commons::metrics;
+use failure::{Error, Fallible};
+use prometheus::{Histogram, IntCounter, IntCounterVec, IntGauge, IntGaugeVec};
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::net::{IpAddr, Ipv4Addr};
+use std::sync::Arc;
+use structopt::StructOpt;
+
+lazy_static::lazy_static! {
+    static ref V1_GRAPH_INCOMING_REQS: IntCounter = register_int_counter!(opts!(
+        "dumnati_pe_v1_graph_incoming_requests_total",
+        "Total number of incoming HTTP client request to /v1/graph"
+    ))
+    .unwrap();
+    static ref UNIQUE_IDS: IntCounter = register_int_counter!(opts!(
+        "dumnati_pe_v1_graph_unique_uuids_total",
+        "Total number of unique node UUIDs (per-instance Bloom filter)."
+    ))
+    .unwrap();
+    static ref ROLLOUT_WARINESS: Histogram = register_histogram!(
+        "dumnati_pe_v1_graph_rollout_wariness",
+        "Per-request rollout wariness.",
+        prometheus::linear_buckets(0.0, 0.1, 11).unwrap()
+    )
+    .unwrap();
+    static ref GRAPH_FINAL_EDGES: IntGaugeVec = register_int_gauge_vec!(
+        "dumnati_gb_scraper_graph_final_edges",
+        "Number of edges in the cached graph, after processing",
+        &["stream"]
+    ).unwrap();
+    static ref GRAPH_FINAL_RELEASES: IntGaugeVec = register_int_gauge_vec!(
+        "dumnati_gb_scraper_graph_final_releases",
+        "Number of releases in the cached graph, after processing",
+        &["stream"]
+    ).unwrap();
+    static ref LAST_REFRESH: IntGaugeVec = register_int_gauge_vec!(
+       "dumnati_gb_scraper_graph_last_refresh_timestamp",
+        "UTC timestamp of last graph refresh",
+        &["stream"]
+    ).unwrap();
+    static ref UPSTREAM_SCRAPES: IntCounterVec = register_int_counter_vec!(
+       "dumnati_gb_scraper_upstream_scrapes_total",
```

Comment:
This and the metrics above containing the `gb_` prefix should be moved to the graph-builder.

Reply:
Moved all gb related logic and scraper logic to the fcos-graph-builder crate.